python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -97,7 +97,7 @@ def _validateGalsimInterpolant(name: str) -> bool:  # noqa: N805
     )
 
     def setDefaults(self):
-        self.kernelSize = 21
+        self.kernelSize = 25
```

Comment:
Can you put a comment above this to the effect of "We use a kernelSize of 25 because ..."?

Reply:
Good point, John. Can you confirm if I got the AP's side of the reasoning correctly?

Reply:
For `i)` I'd say: "fakes injected to match the 12 pixel aperture flux get measured correctly"

Reply:
Done 🙏🏼